modules/ROOT/pages/getting-started.adoc

```diff
@@ -48,28 +49,32 @@ qemu-kvm -m 2048 -cpu host -nographic -snapshot \
 qemu-img create -f qcow2 -b fedora-coreos-qemu.qcow2 my-fcos-vm.qcow2
 qemu-kvm -m 2048 -cpu host -nographic \
 	-drive if=virtio,file=my-fcos-vm.qcow2 \
-	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign
+	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign \
+	-nic user,model=virtio,hostfwd=tcp::2222-:22
 ----
 +
 .Example launching FCOS with virt-install
 [source, bash]
 ----
 virt-install --connect qemu:///system -n fcos -r 2048 --os-variant=fedora31 --import \
 	--graphics=none --disk size=10,backing_store=/path/to/fedora-coreos-qemu.qcow2 \
-	--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/example.ign"
+	--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/example.ign" \
+	--qemu-commandline="-device virtio-net,netdev=user.0 -netdev user,id=user.0,hostfwd=tcp::2222-:22"
```

Comment:
By default, `virt-install` should already be attaching the default network. Is that not the case on your box? But more generally, I think we should let libvirt be aware of the network config here instead of passing it "behind its back" via `--qemu-commandline`.

Reply:
Ahh, I had misunderstood how `virt-install` set up the network previously, thinking it used a similar usermode networking setup like QEMU. Makes a lot more sense to keep the networking set up via libvirt, and just use the IP address shown on the FCOS serial console.